src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -9,6 +9,8 @@
     <DefineConstants>$(DefineConstants);WORKSPACE</DefineConstants>
     <ApplyNgenOptimization>partial</ApplyNgenOptimization>
 
+    <LangVersion>8.0</LangVersion>
```

Comment:
😕 I thought we already set this?

Reply:
This shouldn't be necessary, https://github.com/dotnet/roslyn/blob/c78f1e594997b0dae930551101f690708b4db764/eng/targets/Settings.props#L178